plugins/ckan_operators/datastore_operator.py

```diff
@@ -489,18 +489,22 @@ def clean_date_format(self, input):
             "%d%b%Y": "%Y-%m-%d",
         }
 
-        for format in format_dict.keys():
-            try:
-                input = input.replace("/", "-")
-                datetime_object = datetime.strptime(input, format)
-                output = datetime_object.strftime(format_dict[format])
-                #print("{} format worked with {} - returning {} as {}".format( format, input, format_dict[format], output) )
-                #print("returning: " + output)
-                return output
-            except ValueError as e:
-                #print("Format {} in clean_date_format() doesnt match input string {}".format(format, input))
-                #print(e)
-                pass
+        if input_format:
+            input = input.replace("/", "-")
+            input_format = input_format.replace("/", "-")
```

Comment:
Why we don't assign "%d-%m-%Y" directly to input format?

Reply:
great question! Sometimes we get a `date`, which could be `%d-%m-%Y` but other time we get a `timestamp` which would include hours minutes and seconds

Reply:
right, I see your points!